frontend/integration-tests/tests/alertmanager.scenario.ts

```diff
@@ -15,6 +15,7 @@ import * as horizontalnavView from '../views/horizontal-nav.view';
 import { execSync } from 'child_process';
 
 const replaceInput = async (fieldName: string, value: string) => {
+  await monitoringView.wait(until.elementToBeClickable(firstElementByTestID(fieldName)));
```

Comment:
We shouldn't have a `monitoringView.wait`. We should just call `browser.wait` and use the default timeout. There's nothing specific to the monitoring view in that helper.

Reply:
ok, fixed